public/docs/md/charts/en/group-by.md

```diff
@@ -59,4 +59,14 @@ For formulas *with* a group-by, Amplitude ranks the groups by the largest overal
 
 {{partial:admonition type='note'}}
 If group-by pruning occurs with multiple formula terms combined with operators, formulas may take longer to load because Amplitude runs additional queries to make sure that all formula terms are querying the same groups.
-{{/partial:admonition}}
\ No newline at end of file
+{{/partial:admonition}}
+
+## Group By in Experiment
+
+Everything in this doc so far is for analytics chart excluding experiment results. In Experiment end to end and experiment results, we limit the number of group by groups returned to 10. The rows are sorted by the sum of exposures across all the variants. Some rows may have "(none)" which means the property is missing. See this [doc](docs/faq/unexpected-values-in-user-counts) for more information. The group by is applied to the exposure event.
+
+On the statistics side of things, people may worry about [multiple hypothesis testing](docs/feature-experiment/advanced-techniques/multiple-hypothesis-testing) when using a group by. Amplitude experiment does not do any additional corrections for using a group by because we don't know how much slicing and dicing you are going to do before the analysis. Also, using group bys is more doing exploratory work and it will get very hard to reach stat sig if you have doing lots of hypothesis tests. If you are worried that you might have found a false positive, one way is to split your experiment into to date ranges, and do all the peeking and hypothesis testing you want on one dataset and then see if those results are reproducible on the second date range. You can think of this as similar to doing a [train-test split](https://machinelearningmastery.com/train-test-split-for-evaluating-machine-learning-algorithms/) when you are training a Machine Learning model where you do the hyperparameter tuning on the training set and then evaluate the model on the unseen test set so you get an unbiased error estimate.
```

Comment:
Am I allowed to link to https://machinelearningmastery.com/train-test-split-for-evaluating-machine-learning-algorithms/ or should I link to a more academic article

Reply:
I think it's fine to link to that article. We don't have any guidelines around that.

Reply:
Is experiment "end-to-end" a term we use externally in the product anywhere?